control/server.py

```diff
@@ -59,9 +60,37 @@ class GatewayServer:
         discovery_pid: Subprocess running Ceph nvmeof discovery service
     """
 
+    class RPCGuard:
+        RPC_GUARD_LOCK_TIMEOUT = 300
```

Comment:
@baum the simplification in the code is quite minimal and I'd rather leave it as is and not risk getting stuck in case of a scenario I didn't think of.